test/Chainweb/Test/Utils.hs

```diff
@@ -419,12 +422,13 @@ withChainServer
     => ToJSON t
     => FromJSON t
     => PayloadCas cas
+    => Logger logger
     => ChainwebServerDbs t logger cas
     -> (ClientEnv -> IO a)
     -> IO a
 withChainServer dbs f = W.testWithApplication (pure app) work
   where
-    app = chainwebApplication (Test singletonChainGraph) dbs
+    app = chainwebApplication (Test singletonChainGraph) dbs undefined
```

Comment:
please use `error "undefined called at Chainweb.Test.Utils.withChainServer"` instead. Also consider adding a `HasCallStack` constraint.

Reply:
Oh crap, good catch, that was definitely just an oversight.